pkg/integrations/cloudwatch_exporter/config.go

```diff
@@ -17,10 +17,11 @@ import (
 )
 
 const (
-	metricsPerQuery       = 500
-	cloudWatchConcurrency = 5
-	tagConcurrency        = 5
-	labelsSnakeCase       = false
+	metricsPerQuery                  = 500
+	cloudWatchConcurrency            = 5
+	tagConcurrency                   = 5
+	labelsSnakeCase                  = false
+	defaultDecoupledScrapingInterval = time.Minute * 5
```

Comment:
5m is the default scrape interval in YACE:
https://github.com/nerdswords/yet-another-cloudwatch-exporter/blob/master/cmd/yace/main.go#L133